apps/designer/tsconfig.json

```diff
@@ -1,10 +1,35 @@
 {
-  "extends": "tsconfig/nextjs.json",
-  "include": ["next.d.ts", "**/*.ts", "**/*.tsx"],
```

Comment:
this deletion is not correct. The intended settings should go into packages/tsconfig/base.json or packages/tsconfig/next.json or packages/tsconfig/react-library.json

This works differently for turborepo based monorepos